### PR TITLE
firs pull request

### PR DESCRIPTION
**Pull request by RoyNyaga**
We purposely left comments in the css file for easy review, if the the project is accepted we will remove them.

### CODE REVIEW

**Comment by mosaaleb** [comment]
Reviewing

**Review by mosaaleb — CHANGES_REQUESTED** [review]
### Impressive @RoyNyaga, @R4Ajeti :clap:

#### The UI is clean. Your code base is very well structured and have semantic tags as recommended. Still there are some requirements needs be met before moving on.


#### Nav Bar:

- [ ] Logo alone shouldn't take the full width of the page. The Nav Bar should display the TNW logo at the leftmost part of it.

Screenshot from 2019-07-30 19-26-07
----

- [ ] It should use fixed positioning.
- [ ] The Hamburger (bars) icon should not appear on screens bigger than mobile.

Screenshot from 2019-07-30 19-51-41


#### Latest News section:
- [ ] Each article should have an image on the leftmost part of the article and its title on the right side -- the image should be used as image tag. [Why?](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) 


#### Latest Funcing section:
- [ ] It should have three articles, each one wrapped around a pair of `<article>` -- you have already a div, let's just replace each div with `<section>` tag.

#### From Apps until Distract sections:
- [ ] It should have eight articles, each one wrapped around a pair of `<article>` -- Also you have already done this but inside `<li>` element, let's change it to `<article>` for more semantic tags.
- [ ] Let's rename also the sections as seen on TNW page -- All the section that you have is "Apps".

#### Latest Deals section:
- [ ] Let's rename the last section before footer from "Latest News" to "Latest Deals".


#### General:
- [ ] Let's add [stickler](https://github.com/microverseinc/linters-config/tree/master/css) to make sure that we have good styling guidelines.

- [ ] Let's add the readme file with the it's relevent information.


Please handle these requirements and submit a new code review.

#### Happy coding :computer:

**Comment by RoyNyaga** [comment]
greetings sir, you reviewed my project
https://github.com/RoyNyaga/building_with_responsive_designs/pull/1 .
Thanks for responding this early for the review. I will like us to talk a
little bit about some points you raised. I tried reaching you on slack but
so many people are called Muhammad so i don't know which, please text me on
slack lets talk about my worries.

**Comment by bolah2009** [comment]
Currently under review

File: index.html [hunk 1]
```diff
@@ -68,361 +68,950 @@
...
+                                <li style="color: #555;">$39</li>
+                            </ul>
+                        </section>
+                    
+                        <section class="grid-five-inner-grid">
+                            <a href="#" class="grid-five-picture"></a>
+                            <h4 class="grid-five-header-four"><a href="#">The Ultimate Cisco Networking Expert Certification Training Bundle</a></h4>
+                            <ul class="grid-five-secondary-list">
+                                <li style="text-decoration: line-through; color:#aaa;">$2843.91</li>
+                                <li style="color: #555;">$39</li>
```
Comment (bolah2009) [inline index.html:952]: `SUGGESTION`

Avoid using in-line styles as much as possible. You can style this in your stylesheet.

File: index.html [hunk 2]
```diff
@@ -68,361 +68,950 @@
...
+                            </ul>
+                        </section>
+                    
+                        <section class="grid-five-inner-grid">
+                            <a href="#" class="grid-five-picture"></a>
+                            <h4 class="grid-five-header-four"><a href="#">The Ultimate Cisco Networking Expert Certification Training Bundle<
```
Comment (bolah2009) [inline index.html:970]: `SUGGESTION`

Avoid using in-line styles as much as possible. You can style this in your stylesheet.

File: index.html [hunk 3]
```diff
@@ -68,361 +68,950 @@
...
+                                    An introvert's guide to professional networking
+                                </a>
+                            </h4>
+                        </li>
+                    </ul>
+                </article>
+                <article class="section-direct-child">
+                    <h2 class="section-four-main-heading">
+                        <a href="#">
+                            Insights<i style="position: relative; left: 5px; top: 2px;" class="fas fa-angle-right"></i>
```
Comment (bolah2009) [inline index.html:709]: `SUGGESTION`

Avoid using in-line styles as much as possible. You can style this in your stylesheet.

File: index.html [hunk 4]
```diff
@@ -68,361 +68,950 @@
...
-                    </a>
-                    <div class="grid-three-content-div-main-div">
-                        <a class="grid-three-content-main-div-link" href="#">
-                             <i class="fas fa-tag"></i>
-                            <span>Pharmaceuticals, +2</span>
+                </article>
+                <article class="section-direct-child">
+                    <h2 class="section-four-main-heading">
+                        <a href="#">
+                            Creative<i style="position: relative; left: 5px; top: 2px;" class="fas fa-angle-right"></i>
```
Comment (bolah2009) [inline index.html:561]: `SUGGESTION`

Avoid using in-line styles as much as possible. You can style this in your stylesheet.

File: index.html [hunk 5]
```diff
@@ -68,361 +68,950 @@
...
+                    
+                </article>
+            </section>
+
+            <section id="section-four">
+                <h2 style="display: none;">slkjdf</h2>
+                <article class="section-direct-child">
+                    <h2 class="section-four-main-heading">
+                        <a href="#">
+                            Apps<i style="position: relative; left: 5px; top: 2px;" class="fas fa-angle-right"></i>
```
Comment (bolah2009) [inline index.html:339]: `SUGGESTION`

Avoid using in-line styles as much as possible. You can style this in your stylesheet.

File: index.html [hunk 6]
```diff
@@ -68,361 +68,950 @@
...
-                        <h2 class="grid-one-small-heading" id="grid-one-biggest-heading3">Silk orad side admin extradicted to us hit with 78 months prison sentence</h2>
-                        <ul class="grid-one-list">
-                            <li><a href="#">By Yessi Bello</a></li>
-                            <li><a href="#"> 5 hours ago</a></li>
-                            <li><a href="#">share 0</a></li>
-                        </ul>
+      <main class="main">
+            <section id="grid-one"> <h2 style="display: none;">sss</h2>
+                <article id="grid-one-div-one">
+                    <div style="display: none;" id="displaced-div">
```
Comment (bolah2009) [inline index.html:74]: Why did you have hidden `div` here :thinking: ?

File: index.html (hunk 2 above)
Comment (bolah2009) [inline index.html:1010]: `SUGGESTION`

Consider adding a margin to the sides of this section and implement the positioning of the `% off` section as shown below:

|Current|Preffered|
| :-----: | :------: |
| lastest deal current  | lastest deal preffred |

**Review by bolah2009 — APPROVED** [review]
## Nice Job Team :clap:

The [clone](https://raw.githack.com/RoyNyaga/building_with_responsive_designs/project_layout/index.html) looks good to me and is responsive. :+1: 


#### Check out the comment below and [these errors](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2FRoyNyaga%2Fbuilding_with_responsive_designs%2Fproject_layout%2Findex.html) on using an heading (h1-h6) element in an `<article>` and `<section>` elelment. Consider checking them out before merging.

Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Comment by RoyNyaga** [comment]
Thanks sir will do that before merging

**Pull request merged**